### PR TITLE
Use generic link to Dulwich website

### PR DESCRIPTION
**Pull request by Morganov**
- [x] I provide my work under the [project license](https://github.com/progit/progit2/blob/master/LICENSE.asc).
- [x] I grant such license of my work as is required for the purposes of future print editions to [Ben Straub](https://github.com/ben) and [Scott Chacon](https://github.com/schacon).

## Changes

- Due to errors on loading particular webpages from dulwich.io the HTML checker fails on each build. And it's better to use the main page only instead of the list of pages.

### CODE REVIEW

File: book/B-embedding-git/sections/dulwich.asc [hunk 1]
```diff
@@ -40,5 +40,4 @@ porcelain.log('.', max_entries=1)
 
 ==== Further Reading
 
- * The official API documentation is available at https://www.dulwich.io/docs/api/[].
- * Official tutorial at https://www.dulwich.io/docs/tutorial[] has many examples of how to do specific tasks with Dulwich.
+The API documentation, tutorial, and many examples of how to do specific tasks with Dulwich available on the official website https://www.dulwich.io[].
```
Comment (HonkingGoose) [inline book/B-embedding-git/sections/dulwich.asc:43]:
```suggestion
The API documentation, tutorial, and many examples of how to do specific tasks with Dulwich are available on the official website https://www.dulwich.io[].
```

I think it reads better with the article added.

**Review by HonkingGoose — COMMENTED** [review]
What do you think of this change?

File: book/B-embedding-git/sections/dulwich.asc (hunk 1 above)
Comment (Morganov) [reply book/B-embedding-git/sections/dulwich.asc]: Thanks! Fixed.

**Comment by ben** [comment]
✨

**Pull request merged**